src/components/Slices/SlicesTable.jsx

```diff
@@ -102,23 +103,24 @@ const SlicesTable = ({
 
   // filter out any columns that should be disabled
   columns = columns.filter((column) => !disabled.includes(column.field));
-  return (
-    <MaterialTable
-      options={options}
-      columns={columns}
-      data={slices}
-      detailPanel={[
-        {
-          tooltip: "Address Details",
-          render: (rowData) => (
-            <Box p={1} className={classes.panel}>
-              <SliceDetails client={client} slice={rowData} network={network} />
-            </Box>
-          ),
-        },
-      ]}
-    />
-  );
+  return null;
+  // return (
```

Comment:
@bucko13 What do we want to replace this with? (I'm assuming this package is dead, and doesn't work w new MUI or something). We could open the CL and use that, but maybe not?

Reply:
Good question. Guess we should see what the specific problems are. Maybe it's as easy as just upgrading this library for now. 🤞.